styles.css

```diff
@@ -27,128 +26,275 @@ body {
-  letter-spacing: 0.1px;
-  text-align: center;
-}
-
-h1 {
+.hamburger {
+  display: none;
   font-size: 2rem;
-  text-align: center;
+  color: var(--white-color);
+  margin: 0 2rem;
+  cursor: pointer;
 }
 
-hr {
-  background-color: var(--line-color);
+.navbar {
+  display: flex;
+  align-items: center;
+  height: 100%;
 }
 
-form {
-  width: 100%;
-  text-align: center;
-  display: inline-block;
+.navbar ul {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  margin-left: auto;
 }
 
-input {
-  padding: 12px 20px;
-  margin: 8px 0;
-  display: inline-block;
-  border: 1px solid var(--border-color);
-  box-sizing: border-box;
+.navbar ul li {
+  padding: 0 2rem;
 }
 
-.btn {
-  background-color: var(--primary-color);
+.nav-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  padding: 0 1rem;
   color: var(--white-color);
-  font-weight: normal;
-  cursor: pointer;
+  font-weight: bold;
+  text-decoration: none;
 }
 
-.navbar {
-  position: fixed;
-  transition: background 0.4s ease-in-out;
+.nav-link:hover,
+.active {
+  background-color: var(--active-color);
+}
+
+.video-container {
+  position: relative;
   width: 100%;
-  overflow: hidden;
-  background-color: var(--primary-color);
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+  height: 0;
+  padding-bottom: 56.25%;
+  margin: 2rem 0;
+}
+
+.video-container iframe {
+  position: absolute;
+  width: 100%;
+  height: 100%;
   top: 0;
-  right: 0;
   left: 0;
-  z-index: 1030;
 }
 
-.navbar ul li {
-  padding: 2rem 2rem;
+/* about section */
+#about-section {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 10vh 0;
+  text-align: center;
 }
 
-#navbar ul {
-  list-style: none;
-  display: flex;
+/* download section */
+#download-section {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 10vh 0;
+  text-align: center;
 }
 
-/* Style the navigation menu */
+/* play section */
+#play-section {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 10vh 0;
+  text-align: center;
+}
 
-.nav-link {
-  list-style-type: none;
-  margin: 0;
-  overflow: hidden;
-  background-color: var(--primary-color);
-  color: var(--white-color);
-  display: flex;
+/* play section */
+#contact-section {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 10vh 0;
   text-align: center;
-  float: left;
-  padding: 14px 16px;
-  text-decoration: none;
 }
 
-/* Change the link color to #9d4e4e  on hover */
+.contact-section {
+  background-color: var(--primary-color);
+  padding: 80px 0;
+}
+
+.form {
+  max-width: 400px;
+  margin: 0 auto;
+}
+
+input[type="email"] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+  border: 2px solid #ccc;
+  border-radius: 2px;
+}
+
+input[type="email"]:focus {
+  border: 2px solid var(--primary-color);
+}
 
-.nav-link:hover {
+.btn {
   background-color: var(--active-color);
+  border: none;
+  color: var(--white-color);
```

Comment:
Use CSS variables effectively: The use of CSS variables is a great way to centralize and manage theme attributes. However, it's important to use them effectively and avoid duplication. In some places, like the --white-color variable, the value is the same as the default value, and it's not necessary to define it again. Otherwise greater job on defining and using your variables across the style to avoid duplication.